.github/workflows/build_push.yml

```diff
@@ -11,15 +11,64 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Login to Docker Hub
+      - name: Set defaults networks based on tag
+        id: defaults
         run: |
-          docker login --username ${{ secrets.DOCKER_USERNAME }} --password ${{ secrets.DOCKER_PASSWORD }}
+          case ${GITHUB_REF#refs/*/} in
+            v0.5* | v0.6*)
+                NETWORK=beaconworld
+                SEEDS="08edefb35e6f4f7baeb27303d438ee3daa2e9ace@connect-beaconworld.fetch.ai:36656"
+                RPC_ENDPOINT="https://rpc-beaconworld.fetch.ai:443"
+                LATEST=false
+            ;;
+            v0.2*)
+                NETWORK=agentworld
+                SEEDS="b387f0909e0eac780b8bbe25f04953136b61d0e4@connect-agentworld.prod.fetch-ai.com:26856"
+                RPC_ENDPOINT="https://rpc-agentworld.fetch.ai:443"
+                LATEST=false
+            ;;
+            *)
+                NETWORK=fetchhub
```

Comment:
I like this approach but I think that we need to think about a way to bake more that one network configuration into the image for a given reference or maybe it is simply a new tag of the image which would fit in this model in a more clean way.

In any case we need a way for users to be able to connect to fetchhub or testfetchhub network

Reply:
Yep, that's the hard choice here, hard to tell with the current state of versions and networks. My idea here in the long run was to have default images ready to connect to mainnet out of the box, and provide specific tags for other networks with a matching builtin config. I think this logic here may evolve in a (version + default settings => tag name) kind of matrix so we'd have on docker hub:

```
fetchd:vA.B.C fetchd:latest (default mainnet)
    |_ fetchd:fetchhub (default mainnet)
    |_ fetchd:fetchhubtest (default testnet)
    |_ fetchd:another-testnet (default another-testnet)
fetchd:vD.E.F
    |_ fetchd:another-beta-testnet (default another-beta-testnet)
```